Makefile

```diff
@@ -5,38 +5,34 @@ CCFLAGS = -Wall -c
 
 ifeq ($(OS),Windows_NT)
 	os = windows
-	CCFLAGS += -D WIN32
+	CCFLAGS += -D WIN32 -D IA32
```

Comment:
Oh this Makefile is just grabbed out from my stock. Thanks for pointing them out.